.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Test and update
+
+on:
+  schedule:
+    - cron: "37 6 * * *"
+  workflow_dispatch: {}
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    if: github.repository == 'williamkapke/node-compat-table'
```

Comment:
This is so that any existing forks of this repo don't accidentally run this action, which will break their forks.

Reply:
Once a fork gets this workflow, if they have actions enabled (new forks don't by default, old forks do), this workflow will run and commit to their repo. Since this happens on both, the fork and upstream will diverge, meaning the two can no longer be synced. I've hit this many times before (most recently on DefinitelyTyped and TypeScript's scheduled workflows).

Reply:
If they have the same results though, wouldn't the built-in sync feature Just Work?

Reply:
GitHub's built-in sync button syncs to the fork's default branch, so this workflow would be pulled in, and would be present on any new fork (though new forks have actions off by default, but can be turned back on).

Also, I always keep my fork's default branch up to date if I'm pulling locally (which is the same as GitHub's sync). Way way easier to work locally when main is up to date.

Reply:
The easiest way to work locally is to add the source as a remote, delete the master/main branch *entirely*, and always use `source/main` instead of having to keep a local one up to date.